### ISSUE TITLE
post-install-script02.ps1 throws error message confusing participants

### ISSUE THREAD

**Issue by JuergenSchwertl** [microsoft/MCW-Securing-Azure-IoT-solutions#26]
manually running post-install-script02.ps1 throws an error message on [line 50](https://github.com/microsoft/MCW-Securing-Azure-IoT-solutions/blob/master/Hands-on%20lab/Scripts/post-install-script02.ps1#L50) complaining about the ...vmcx file not being available (which is correct at that stage as it's being created afterwards). That error message should be suppressed as it confuses participants.

Suggest to suppress error message by adding "-ErrorAction SilentlyContinue" or in short:
```powershell
$item = get-item "C:\VMs\UBUSRV\Virtual Machines\BE674C9C-0461-4F44-B105-6893F5618F46.vmcx" **-ea silentlycontinue** 
```

**Comment by givenscj** [comment]
Excellent suggestion, added!

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #28
